CMakeLists.txt

```diff
@@ -239,4 +239,15 @@ add_custom_target(check-clang-tidy-p0
         "src/primer/p0_trie.cpp"
         )
 
+add_custom_target(check-clang-tidy-p1
+        ${BUSTUB_BUILD_SUPPORT_DIR}/run_clang_tidy.py                     # run LLVM's clang-tidy script
+        -clang-tidy-binary ${CLANG_TIDY_BIN}                              # using our clang-tidy binary
+        -p ${CMAKE_BINARY_DIR}                                            # using cmake's generated compile commands
+        -j 1
```

Comment:
Can we remove `-j1` here? I can't remember why I added it last time, but we absolutely don't want clang-tidy to run on only one core.